src/components/Main/Main.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { usePokemon } from '../../hooks/pokemon.js';
+import Select from '../Select/Select.js';
+import Search from '../Search/Search.js';
+
+import './Main.css';
+
+export default function Main() {
+  const { pokemon, types, handleTypeChange, handleSearch, setSearchTerm, searchTerm, loading } =
+    usePokemon();
+  if (loading) return <div className="loader"></div>;
+  return (
+    <div>
+      <Search {...{ handleSearch, setSearchTerm, searchTerm }} />
+      <Select types={types} handleTypeChange={handleTypeChange} />
+      {searchTerm !== '' && <div>Searching by {searchTerm}</div>}
+      <>
```

Comment:
you only need these if you are writing a separate component and need it to return one thing. in this case it's not doing anything.

but you should probably move all this stuff over to a separate component, so you would need the <>